is.go

```diff
@@ -29,7 +29,7 @@ func Is(err error, v Vice) bool {
 	case Canceled:
 		return isCanceled(err)
 	default:
-		return false
+		return isNoVice(err)
```

Comment:
can you make this its own case, instead? 

that way, if you do something like `vice.Is(err, vice.Vice(1235324))` it will always be false. but `NoVice` is a special case for testing.

Reply:
Good point! Will do.